README.md

```diff
@@ -12,29 +12,43 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
 
 ## Get Started
 
-### Prerequisite 
+### Prerequisite
+
 - [Python interpreter](https://docs.python.org/3/using/index.html)
 - Cookiecutter
   - Mac: `brew install cookiecutter`
   - Ubuntu: `brew install cookiecutter`
 - Go version >= 1.15
 
 ## Usage
+
 - Download **latest** version of gin-templates.
   ```
   go get github.com/nimblehq/gin-templates
   ```
+
 - Build the binary file.
   ```
-  go build -i -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
+  go build -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
   ```
-- Run create the project command.
+
+- Create the project from a template on a **main** branch.
```

Comment:
English-wise, it still could be better:

```suggestion
- Create the project using gin-templates. Note that the **main** branch is used.
```

Later on, when the wiki is added, the content could be enhanced with thee following:

```suggestion
- Create the project using gin-templates. Note that the **main** branch is used by default. Refer to [this wiki page](REPLACE WITH LINK TO WIKI PAGE) for instructions on how to use a different branch.
```

Reply:
Fixed in f025cbc28da3533a347467ed4e806b511ee97c7c